qa-tests-backend/src/test/groovy/VulnScanWithGraphQLTest.groovy

```diff
@@ -145,6 +146,8 @@ class VulnScanWithGraphQLTest extends BaseSpecification {
     }
 
     @Unroll
+    // TODO(ROX-29221): Fix the test for fixable image info from CVEID
+    @IgnoreIf({ Env.get("ROX_FLATTEN_CVE_DATA", null) == "true" })
```

Comment:
```suggestion
    @IgnoreIf({ Env.get("ROX_FLATTEN_CVE_DATA") == "true" })
```

I think it will be better to have a method that return boolean instead of string

Reply:
Looks like Groovy's `==` works similar to Java's `.equals()` . So `==` should be fine here.
https://groovy-lang.org/style-guide.html#_equals_and